ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/agent/MLAgentExecutor.java

```diff
@@ -704,6 +700,15 @@ private org.opensearch.ml.common.contextmanager.ContextManager createToolsOutput
         manager.initialize(config != null ? config : new HashMap<>());
         return manager;
     }
+    /**
+     * Create SlidingWindowManager
+     */
+    private org.opensearch.ml.common.contextmanager.ContextManager createSlidingWindowManager(Map<String, Object> config) {
```

Comment:
[nit] `org.opensearch.ml.common.contextmanager.` --> we can add this in the import